patterns/1-initial/scoring-grid.md

```diff
@@ -0,0 +1,114 @@
+|       CODEOWNERS       | No CODEOWNERS file; ownership of repo unclear, or ownership belongs to a defunct team; contact details for team are outdated or incorrect.                 | CODEOWNERS file exists; named individuals required for review; main branch locked down; requires at least one person to provide PR review. | CODEOWNERS file exists; named team required for review; main branch locked down requiring PR review                                     | CODEOWNERS file exists; named team required for review, main branch locked down requiring PR review; accurate team contact details available in README.                                                   |
+| CONTINUOUS INTEGRATION | No Continuous Integration setup; builds are manual, and are transferred directly from developer laptop to production environment. Main branch fails tests. | Continuous Integration exists; requires manual triggers. Process not checked into source control. No PR integration.                       | Continuous Integration is well integrated; PRs automatically checked and verified. Process scripted as part of code base.               | Continuous Integration is fully automated; high trust test suite; features on main branch are automatically rolled out to production; use of canary builds; feature toggles.                              |
+
+Apply the grid against one or more repositories; deciding on a score. This can be done indiviually using an expert, or as team as a retrospective discussion.
+
+Based on the score, look to the next column, and identify actions that would lead to an improved grade.
+
+Agree with the team to rescore on a regular basis, say weekly, monthly, quarterly based on available capacity.
+
+Make time for teams to action improvements; ideally make the work part of their normal repsonsibilities, visible on shared work boards as work tickets.
+
+## Resulting Context
+
+- Scoring Grid communicated to all engineering teams and individual contributors
+- Implicit understanding of "What good looks like?" at the repository / project level
+- Improved perception of Engineering Leadership from Engineers
+- Incremental prioritisation of tech debt aimed at improving the quality of code and code bases
+- Possible automation of scoring grid; applied daily/nightly to all codebases
+- Possible "competitive ranking" of code bases, leading to rewards for following best practice
+
+## Known Instances
+
+Where has this been seen before?
+
+- Company A - 100 strong co-located engineering team with well established InnerSource community - used as guidance for all teams, and published as part of internal InnerSource website - exact grid not published in this pattern
```

Comment:
```suggestion
- Company A (100 strong co-located engineering team with well established InnerSource community): Used as guidance for all teams, and published as part of internal InnerSource website (exact grid not published in this pattern)
```

Suggesting a slightly different format for this sentence (also using the same format for the next sentence).